test/acceptance/Dockerfile.allure

```diff
@@ -0,0 +1,10 @@
+FROM quay.io/pmacik/yhxue911-python-allure
+
+RUN echo -e "#!/bin/bash\n\
+    if [ \$1 == \"generate\" ]; then allure generate /allure/results -o /allure/report --clean; fi\n\
+    if [ \$1 == \"serve\" ]; then allure serve -p 8080 /allure/results; fi" >> /entrypoint.sh && chmod +x /entrypoint.sh
```

Comment:
Instead of crafting `entrypoint.sh` during the build, please add it to the repo under `test/acceptance/entrypoint.sh` - then you need to do only:

```
COPY entrypoint.sh /entrypoint.sh
```

Reply:
done